tasks/create-pyxis-image/create-pyxis-image.yaml

```diff
@@ -126,8 +105,7 @@ spec:
             DIGEST="${CONTAINER_IMAGE##*@}"
             PULLSPEC="${REPOSITORY}@${DIGEST}"
             MEDIA_TYPE=$(skopeo inspect --raw "docker://${PULLSPEC}" | jq -r .mediaType)
-            TAGS=$COMMON_TAGS
-            if [ $(jq ".components[${i}].tags | length" "${SNAPSHOT_SPEC_FILE}") -gt 0 ] ; then
+            if [ $(jq ".components[${i}].tags | length" "${SNAPSHOT_SPEC_FILE}") -gt 0 ] ; then # TODO
```

Comment:
@mmalina would love to get your opinion here. Before, you had two methods of providing tags, and if you chose neither, the task would fail. Now we just have one... should I fail the task if this if statement is false?

Reply:
Hmm, good question. First I was like, maybe we should just hardcode a default tag here. But then you would need the same thing in the other places - like rh-sign-image and push-image. So if we do want a default in case the mapping doesn't specify anything at all for a component, it should be injected at the same place where we expand the variables - apply-mapping. And all the other tasks should just fail if they encounter a component with no tag at all. It might make sense to just skip them, but I anticipate one day we would end up with missing components from a push and it would be hard to debug what happened. So with or without a default tag added in apply-mapping - I think the subsequent tasks should fail if they encounter a component with no tag. WDYT?

Reply:
I am good with creating a ticket to fail apply-mapping if a component has no tags and there are no default tags. I think this would also remove the need for the when clauses in the pipeline definitions (which is currently when commonTags != ""). I am not sure about an if to fail the task here though. We just trust many things are already in each component entry of the snapshot (containerImage, repository) so we should probably treat tags the same, no?

Reply:
>  We just trust many things are already in each component entry of the snapshot (containerImage, repository) so we should probably treat tags the same, no?

That's a good point. So yeah, please create a ticket to fail in apply-mapping if there is no tag in the resulting mapping and then we don't need this check here (you can remove it already).

Reply:
https://issues.redhat.com/browse/RELEASE-932 - so the PR that updates push-snapshot to no longer support data.images will remove commonTags from the task and the pipeline definitions (so the issue in RELEASE-50 will then be an issue again) but then with this one it will once again be fixed. Resolving this